### PR TITLE
#rfc hardcoded static assets

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
Signed-off-by: Dzmitry Lahoda <dzmitry@lahoda.pro>

## Issue

How we can release assets.

### CODE REVIEW

File: rfcs/0012-hardcoded-static-assets.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+RPC and extrinsic check runtime configured asset state before using 
+hardcode.
+
+Foreign locations and symbols will not change.
+
+## Who can update hardcoded assets?
+
+The root can update hardcoded assets. Such assets are permissioned.
+
+## Who uses hardcoded assets?
```
Comment (vimukthi-git) [inline rfcs/0012-hardcoded-static-assets.md:42]: Could you link to the place in code they hard code? Just want to see an example of what this means in the runtime.

**Review by KaiserKarel — APPROVED** [review]
Agree, @andyjsbell and @vimukthi-git have the final call from me on this, but discussed it in DMs too and think it is very reasonable and reduces operational load during launch.

File: rfcs/0012-hardcoded-static-assets.md (hunk 1 above)
Comment (dzmitry-lahoda) [reply rfcs/0012-hardcoded-static-assets.md:42]: currency.rs in https://github.com/ComposableFi/composable/pull/2118

Comment (PoisonPhang) [reply rfcs/0012-hardcoded-static-assets.md:42]: code/parachain/runtime/primitives/src/currency.rs

**Review by vimukthi-git — APPROVED** [review]
Prefer if we follow some standard format for these docs where we layout the implementation better. Anyway LGTM for now.

**Review by andyjsbell — COMMENTED** [review]
lgtm

**Pull request merged**